src/main/java/de/tum/in/www1/artemis/domain/exam/event/ExamLiveEvent.java

```diff
@@ -81,11 +80,5 @@ public void setStudentExamId(Long studentExamId) {
         this.studentExamId = studentExamId;
     }
 
-    protected void populateDTO(ExamLiveEventDTO dto) {
-        dto.setId(getId());
-        dto.setCreatedBy(getCreatedBy());
-        dto.setCreatedDate(getCreatedDate());
-    }
-
-    public abstract ExamLiveEventDTO asDTO();
+    public abstract de.tum.in.www1.artemis.web.rest.dto.examevent.ExamLiveEvent asDTO();
```

Comment:
Why did you change the name from `ExamLiveEventDTO` to `ExamLiveEvent`?
Imo the entity classes and DTOs should be quite easily distinguishable in code, since they are used for two different purposes.

Reply:
ExamLiveEventDTO has been renamed because it is an interface.

There is also [another discussion](https://github.com/ls1intum/Artemis/pull/8571#discussion_r1615315172) that suggests naming the interfaces for DTOs SomethingBaseDTO:
> How about calling the Interface `ExerciseBaseDTO` and then the DTO `ExerciseDTO` - so interfaces are always the Base for an DTO.

Feel free to join the discussion and to make suggestions :)